Update connect-to-remote-aadj-pc.md (#8954)
MicrosoftDocs/windows-itpro-docs: windows/client-management/connect-to-remote-aadj-pc.md

- Windows 10

> From its release, Windows 10 has supported remote connections to PCs joined to Active Directory. Starting in Windows 10, version 1607, you can also connect to a remote PC that is [joined to Azure Active Directory (Azure AD)](hhttps://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-join). Starting in Windows 10, version 1809, you can [use biometrics to authenticate to a remote desktop session](https://docs.microsoft.com/windows/whats-new/whats-new-windows-10-version-1809#remote-desktop-with-biometrics).

Reviewer: ```suggestion
From its release, Windows 10 has supported remote connections to PCs joined to Active Directory. Starting in Windows 10, version 1607, you can also connect to a remote PC that is [joined to Azure Active Directory (Azure AD)](https://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-join). Starting in Windows 10, version 1809, you can [use biometrics to authenticate to a remote desktop session](https://docs.microsoft.com/windows/whats-new/whats-new-windows-10-version-1809#remote-desktop-with-biometrics).
```

![Remote Desktop Connection client](images/rdp.png)

## Set up

- Both PCs (local and remote) must be running Windows 10, version 1607 or later. Remote connections to an Azure AD-joined PC running earlier versions of Windows 10 are not supported.

> - Your local PC (where you are connecting from) must be either Azure AD joined or Hybrid Azure AD joined if using Windows 10 version 1607 and above, or [Azure AD registered](https://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-register) if using Windows 10 version 2004 and above. Remote connections to an Azure AD joined PC from an unjoined device or a non-Windows 10 device are not supported. 

Reviewer: ```suggestion
- Your local PC (where you are connecting from) must be either Azure AD-joined or Hybrid Azure AD-joined if using Windows 10, version 1607 and above, or [Azure AD registered](https://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-register) if using Windows 10, version 2004 and above. Remote connections to an Azure AD-joined PC from an unjoined device or a non-Windows 10 device are not supported. 
```

Author: There are no "-" for Azure AD joined and hybrid Azure AD joined. They are not hyphenated terms

Reviewer 2: OK, so maybe a moderate suggestion, containing only the comma suggestions, will be OK instead:
```suggestion
- Your local PC (where you are connecting from) must be either Azure AD joined or Hybrid Azure AD joined if using Windows 10, version 1607 and above, or [Azure AD registered](https://docs.microsoft.com/azure/active-directory/devices/concept-azure-ad-register) if using Windows 10, version 2004 and above. Remote connections to an Azure AD joined PC from an unjoined device or a non-Windows 10 device are not supported. 
```

  3. If the user who joined the PC to Azure AD is the only one who is going to connect remotely, no additional configuration is needed. To allow additional users or groups to connect to the PC, you must allow remote connections for the specified users or groups. Users can be added either manually or through MDM policies:
     
      - Adding users manually
   
        You can specify individual Azure AD accounts for remote connections by running the following PowerShell cmdlet:
        ```powershell
        net localgroup "Remote Desktop Users" /add "AzureAD\the-UPN-attribute-of-your-user"
        ```
        where *the-UPN-attribute-of-your-user* is the name of the user profile in C:\Users\, which is created based on the DisplayName attribute in Azure AD.

        This command only works for AADJ device users already added to any of the local groups (administrators).
        Otherwise this command throws the below error. For example:
        - for cloud only user: "There is no such global user or group : *name*"
        - for synced user: "There is no such global user or group : *name*" </br>

         > [!NOTE]

>          > For devices running Windows 10, version 1703 or earlier, the user must to sign in to the remote device first before attempting remote connections.

Reviewer: ```suggestion
         > For devices running Windows 10, version 1703 or earlier, the user must sign in to the remote device first before attempting remote connections.
```

         >

>          > Starting Windows 10, version 1709, you can add other Azure AD users to the **Administrators** group on a device in **Settings** and restrict remote credentials to **Administrators**. If there is a problem connecting remotely, make sure that both devices are joined to Azure AD and that TPM is functioning properly on both devices.

Reviewer: ```suggestion
         > Starting in Windows 10, version 1709, you can add other Azure AD users to the **Administrators** group on a device in **Settings** and restrict remote credentials to **Administrators**. If there is a problem connecting remotely, make sure that both devices are joined to Azure AD and that TPM is functioning properly on both devices.
```

      - Adding users using policy
     

>          Starting Windows 10 version 2004, you can add users or Azure AD groups to the Remote Desktop Users using MDM policies as described [in this article](https://docs.microsoft.com/azure/active-directory/devices/assign-local-admin#manage-administrator-privileges-using-azure-ad-groups-preview)

Reviewer: ```suggestion
         Starting in Windows 10, version 2004, you can add users or Azure AD groups to the Remote Desktop Users using MDM policies as described in [How to manage the local administrators group on Azure AD joined devices](https://docs.microsoft.com/azure/active-directory/devices/assign-local-admin#manage-administrator-privileges-using-azure-ad-groups-preview).
```

         > [!TIP]
         > When you connect to the remote PC, enter your account name in this format: AzureAD\yourloginid@domain.com.

>          > [!Note]

Reviewer: ```suggestion
         > [!NOTE]
```

>          > If you cannot connect using Remote Desktop Connection 6.0, you must turn off the new features of RDP 6.0 and revert back to RDP 5.0 by making a few changes in the RDP file. See the details in the [support article](https://support.microsoft.com/help/941641/remote-desktop-connection-6-0-prompts-you-for-credentials-before-you-e).

Reviewer: ```suggestion
         > If you cannot connect using Remote Desktop Connection 6.0, you must turn off the new features of RDP 6.0 and revert back to RDP 5.0 by making a few changes in the RDP file. See the details in this [support article](https://support.microsoft.com/help/941641/remote-desktop-connection-6-0-prompts-you-for-credentials-before-you-e).
```

Reviewer 2: If this had not been a Microsoft Docs document, something more like a personal document, I might have suggested to include the word "this" in the link description. But it is what it is. 🙂

## Supported configurations

> The table below lists out the supported configurations for remotely connecting to an Azure AD-joined PC:

Reviewer: ```suggestion
The table below lists the supported configurations for remotely connecting to an Azure AD-joined PC:
```

> | Criteria | RDP from Azure AD registered device| RDP from Azure AD joined device| RDP from Hybrid Azure AD joined device |

Reviewer: ```suggestion
| Criteria | RDP from Azure AD registered device| RDP from Azure AD-joined device| RDP from hybrid Azure AD-joined device |
```

Author: Not hyphenated terms

Reviewer 2: Suggesting only the casing correction instead:
```suggestion
| Criteria | RDP from Azure AD registered device| RDP from Azure AD joined device| RDP from hybrid Azure AD joined device |
```

| - | - | - | - |

> | **Client operating systems**| Windows 10 2004 and above| Windows 10 1607 and above | Windows 10 1607 and above |

Reviewer: ```suggestion
| **Client operating systems**| Windows 10, version 2004 and above| Windows 10, version 1607 and above | Windows 10, version 1607 and above |
```
